domains/yagnik.is-cool.dev.json

```diff
@@ -1,9 +1,10 @@
 {
   "owner": {
     "username": "yaagnik-patel",
-    "email": "yagnikpatel24818@gmail.com"
+    "email": "yagu2482004@gmail.com"
   },
   "records": {
-    "CNAME": "portfolio-git-main-yaagnik-patels-projects.vercel.app"
+    "CNAME": "portfolio-git-main-yaagnik-patels-projects.vercel.app.",
+    "TXT": "vc-domain-verify=yagnik.is-a.dev,a847575435db916293d3"
```

Comment:
Invalid Vercel site verification file name. See dos for more information: https://docs.is-a.dev/guides/vercel Reopen PR when fixed.